docs/datasources/druid/README.md

```diff
@@ -0,0 +1,228 @@
+# Druid Setup Guide
+
+## Step 1: Add the Druid cluster configuration to `docker-compose.dev`
+
+```
+services:
+  # ...previous services
```

Comment:
the instruction is very hard to run. Please create a standalone example of druid docker-compose and attach its network to `cubejs` container in dev